spec.html

```diff
@@ -32725,7 +32697,7 @@ <h2>Syntax</h2>
 
           TemporalSign :::
             ASCIISign
-            &lt;MINUS&gt;
+            &lt;U+2212 MINUS SIGN&gt;
```

Comment:
This is rejected by esmeta, but I don't know what to put in [esmeta-ignore.json](https://github.com/tc39/ecma262/blob/main/esmeta-ignore.json) to suppress the issue (or even if that is possible, cf. https://github.com/tc39/ecma262/actions/runs/5273991103/jobs/9538007203?pr=3098 and https://github.com/es-meta/esmeta/issues/147 ).